src/App.js

```diff
@@ -1,16 +1,58 @@
-import React from 'react';
+import React from "react";
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList';
+
+const todo = [
+  {
+    task: "Organize Garage",
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: "Bake Cookies",
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state =  {
+      todo,
+      task: '',
+      id: Math.random(),
+    };
```

Comment:
I would omit having the key id with the value Math.random() here to create the ids and use the Date.now() method inside the setState() method in your formSubmitHandler. While the odds are very low of getting a duplication in the id's with it, it is possible.